### PR TITLE
Prefix type name with 'compute-' namespace.

### LINKED ISSUE #52: Prefix type name in compute resources.
`network` and `subnet` resources should be prefixed with `compute-`.

### PR DESCRIPTION
**Pull request by sshyub**
Fixes #52

### CODE REVIEW

File: src/main/java/gyro/google/compute/NetworkResource.java [hunk 1]
```diff
@@ -109,12 +111,22 @@ public void setId(String id) {
...
 
+    @Output
+    public String getSelfLink() {
+        return selfLink;
+    }
+
+    public void setSelfLink(String selfLink) {
+        this.selfLink = selfLink;
+    }
+
```
Comment (deepanjan90) [inline src/main/java/gyro/google/compute/NetworkResource.java]: This isn't related to PR.
This should be part of a different PR stating the need to add this.

File: src/main/java/gyro/google/compute/NetworkResource.java [hunk 2]
```diff
@@ -109,12 +111,22 @@ public void setId(String id) {
...
     @Override
     public void copyFrom(Network network) {
         setId(network.getId().toString());
         setRoutingMode(network.getRoutingConfig().getRoutingMode());
         setDescription(network.getDescription());
         setName(network.getName());
+        setSelfLink(network.getSelfLink());
```
Comment (deepanjan90) [inline src/main/java/gyro/google/compute/NetworkResource.java]: Same as above.

File: src/main/java/gyro/google/compute/package-info.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@DocGroup("Compute")
+package gyro.google.compute;
+
+import gyro.core.resource.DocGroup;
```
Comment (deepanjan90) [inline src/main/java/gyro/google/compute/package-info.java]: This isn't related to PR.
This should be part of a different PR stating the need to add this.

File: src/main/java/gyro/google/compute/NetworkResource.java (hunk 1 above)
Comment (sshyub) [reply src/main/java/gyro/google/compute/NetworkResource.java]: Sure I'll remove this as it's already covered in #51

File: src/main/java/gyro/google/compute/NetworkResource.java (hunk 2 above)
Comment (sshyub) [reply src/main/java/gyro/google/compute/NetworkResource.java]: Sure I'll remove this as it's already covered in #51

File: src/main/java/gyro/google/compute/package-info.java (hunk 3 above)
Comment (sshyub) [reply src/main/java/gyro/google/compute/package-info.java]: Sure I'll remove this.

**Comment by deepanjan90** [comment]
That sounds good.
